attack/scripts/bashrc

```diff
@@ -104,16 +104,16 @@ starting_point() {
     return 1
   fi
   # Identify the starting point mode
-  MODE="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | ."starting-point".mode')"
+  MODE="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | .startingPoint.mode')"
 
   #Determine if mode is internal-instance.
   if [[ "$MODE" == "internal-instance" ]]; then
-    KUBECTL_ACCESS="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | ."starting-point"."kubectl-access"')"
+    KUBECTL_ACCESS="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | .startingPoint.kubectlAccess')"
 
     if [[ "$KUBECTL_ACCESS" == "false" ]]; then
-      ssh "$INTERNAL_HOST_IP" mv ~/.kube /var/local/
-    elif [[ ! -d ~/.kube ]]; then
-      ssh "$INTERNAL_HOST_IP" mv /var/local/.kube ~/
+      ssh "$INTERNAL_HOST_IP" 'ls -d ~/.kube >/dev/null 2>&1 && mv ~/.kube /var/local/'
```

Comment:
can we use the `compgen` trick here instead?  looks neater

Reply:
I know compgen isn't exactly secret, but its also not that commonly used.  So for readability/on going maintaince I think avoiding this type of approach and leaving as is a better approach.

Reply:
We had a protracted conversation about this on a previous pr with @sublimino too and concluded compgen was the neatest solution. Summary:

* Doesn't require redirections
* Reduces magic [ sigils in tests
* Avoids an extra subprocess as it's a built-in

To counter your concern:

* If you don't know it: typing `help compgen` gives a surprisingly succinct and easily comprehensible explanation (as far as bash docs go!)
* We're already standardising towards compgen

Reply:
humour me here.

For this case, the original code has
`ssh "$INTERNAL_HOST_IP" mv ~/.kube /var/local/`

whilst with compgen it now has 
`ssh "$INTERNAL_HOST_IP" '[[ $(compgen -d ~/.kube) ]] && mv ~/.kube /var/local/'`

So, just by referring to your summary point:

- There is no redirection in orignal code
- Seem to have gained, rather than reduced [[
- Seem to have gain a subprocess, whereas original had nope

So I don't see the added value here.  Am I missing something?